lib/components/LoggedInAs.js

```diff
@@ -67,18 +68,33 @@ var LoggedInAs = React.createClass({
   },
 
   renderChooseDevices: function() {
+    var title = '';
     var uploadInProgress = this.props.isUploadInProgress;
+    var isDisabled = uploadInProgress;
 
-    if(this.props.isClinicAccount){
+    if (this.props.isClinicAccount) {
       return null;
     }
+
+    if (_.isEmpty(this.props.targetUsersForUpload) ||
+        this.props.targetUsersForUpload.length < 1) {
```

Comment:
Looks like we initialize `targetUsersForUpload` as an empty array, so I think just one of these checks should be sufficient, no? My preference would be to keep the `_.isEmpty`.